azkaban-hadoop-security-plugin/build.gradle

```diff
@@ -1,12 +1,20 @@
 apply plugin: 'distribution'
 
+configurations {
+    testCompile.extendsFrom compileOnly
```

Comment:
I don't want to duplicate each dependency. IMO, this is better than maintaining 2 duplicate dependency lists. That exposes us to cases like what if somebody adds it to one and not the other for example.